.golangci.yml

```diff
@@ -7,7 +7,7 @@
 
 service:
   # When updating this, also update bin/linters.sh accordingly
-  golangci-lint-version: 1.16.x # use the fixed version to not introduce new linters unexpectedly
+  golangci-lint-version: 1.18.x # use the fixed version to not introduce new linters unexpectedly
```

Comment:
I think this needs to be propagated to common-files.

Reply:
istio/istio has diverged from common files I think. We should update it there as well though